src/BuildASub.java

```diff
@@ -60,4 +68,43 @@ public static String chooseMeat() throws Exception {
             throw new Exception("Meat not set!");
         }
     }
+
+    public static String ChooseCheese(){
+        String SelectedCheese = "";
+        System.out.println("What cheese do you want: \n 1. Regular \n 2. Peppered");
+        Scanner scan = new Scanner(System.in);
+        int CheeseOption = scan.nextInt();
+        switch (CheeseOption){
+            case 1:
+                SelectedCheese = "Regular";
+                        break;
+            case 2:
+                SelectedCheese = "Peppered";
+                        break;
+            default:
+                SelectedCheese ="No Cheese";
+                        break;
+        }
+        return SelectedCheese;
+    }
+
+
+
+    public static Boolean DoYouWantToasted(){
+        Boolean toast = false;
+        System.out.println("Would you like your sub toasted? Y/N : ");
+        Scanner scan = new Scanner(System.in);
+        String EnteredOption = scan.nextLine();
+
+        switch(EnteredOption){
```

Comment:
another option could have been to upper case all the input `EnteredOption.toUpperCase();`